modules/apps/portal-template/portal-template-engine-api/src/main/java/com/liferay/portal/template/engine/TemplateContextHelper.java

```diff
@@ -915,31 +915,13 @@ public byte[] URLtoByteArray(Options options) throws IOException {
 
 		@Override
 		public byte[] URLtoByteArray(String location) throws IOException {
-			if (isLocationAccessDenied(location)) {
```

Comment:
@istvansajtos The commit says "simplify further" while there are only removals. Are these no longer needed? If so, could you update the commit message to state this point and maybe explain a bit?

Reply:
@dantewang , sorry for the confusion. I've updated the commit message to highlight that the very same check is done in URLto*(Options options) methods where we end up eventually.